scripts/scil_tractogram_project_map_to_streamlines.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Projects metrics extracted from a map onto the endpoints of streamlines.
+
+The default options will take data from a nifti image (3D or ND) and
+project it onto the endpoints of streamlines.
+"""
+
+import argparse
+import logging
+
+from dipy.io.streamline import save_tractogram, StatefulTractogram
+
+from scilpy.io.image import load_img
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_reference_arg,
+                             add_verbose_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+from scilpy.image.volume_space_management import DataVolume
+from scilpy.tractograms.streamline_operations import (
+    project_metric_to_streamlines)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+
+    # Mandatory arguments input and output tractogram must be in trk format
+    p.add_argument('in_tractogram',
+                   help='Fiber bundle file.')
+    p.add_argument('out_tractogram',
+                   help='Output file.')
+    p.add_argument('--in_metric', nargs='+', action='append', required=True,
```

Comment:
Comment 1) I would simply say --in_maps. They are not always metrics for everybody necessarily, are they? Can we think of cases where this is confusing?
And in the help, explain that they should be 3D nifti_files? What happens if the file is 4D? Do you upload them as array?

Comment 2) I just want to make sure that the action append is really what you wanted. The nargs='+' already allows you to send `--in_metric file1 file2 file3`. With action append, you can now do:
 `--in_metric file1 --in_metric file2.` Is this what you intended?

Reply:
Comment #1: If the data is 4D I upload them as a list of 1D arrays. Kind of a hack but works?  Is this okay?

Comment #2 I will change the arg name in the next version

Reply:
I don't know who would use it. I guess it's fine! But I would explain it in the help.